### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by ngreenwald**
Great improvements, some of the best

### CODE REVIEW

File: segmentation/evaluation/evaluate_segmentation.py [hunk 1]
```diff
@@ -4,181 +4,140 @@
...
 
 importlib.reload(plot_utils)
 
 
 # code to evaluate accuracy of different segmentation contours
 
 # read in TIFs containing ground truth contoured data, along with predicted segmentation
-base_dir = '/Users/noahgreenwald/Documents/Grad_School/Lab/Segmentation_Project/'
-
-deep_direc = base_dir + 'analyses/20190917_naming/'
```
Comment (ngreenwald) [inline segmentation/evaluation/evaluate_segmentation.py]: Why did you delete these files?

Comment (ngreenwald) [reply segmentation/evaluation/evaluate_segmentation.py]: I don't think they were necessary

**Comment by mapazarr** [comment]
> Great improvements, some of the best
Great explanation, some of the best :-)
